flink-runtime/src/test/java/org/apache/flink/runtime/state/TestTaskStateManager.java

```diff
@@ -46,6 +46,7 @@
 public class TestTaskStateManager implements TaskStateManager {
 
 	private long reportedCheckpointId;
+	private long notifiedCompletedCheckpointId;
```

Comment:
Having a `List<Long>` instead of `long` here would make it useful in a broader set of cases.

Reply:
I think just retaining the last notified completed checkpoint id should be enough like previous `reportedCheckpointId` did.

Reply:
Yes, I agree it's enough currently.